client/reader.js

```diff
@@ -74,18 +74,26 @@ rl.on('line', (line) => {
     }
   } else if (line.indexOf('FLEX: ') > -1) {
     address = line.match(/FLEX:.*?\[(\d*?)\] /)[1].trim();
-    if (line.match( /( ALN | GPN | NUM)/ )) {
-      message = line.match(/FLEX:.*?\[\d*\] ... (.*?)$/)[1].trim();
-      trimMessage = message;
-    } else {
-      message = false;
-      trimMessage = '';
+      if (line.match( /1600\/2\/F\/A/ )) {
```

Comment:
For these, I'd probably want to match it a little wider in case people have different baud rates etc.

```suggestion
      if (line.match( / [0-9]{4}\/[0-9]\/F\/. / )) {
```

This should be enough to match the pattern without false positives. I've also added spaces on either side of the string, to further reduce false positives